### PR TITLE
T-109 On header highlight current page link + add anchors

### PR DESCRIPTION
**Pull request by Bohdan-R**

### CODE REVIEW

File: components/About/About.js [hunk 1]
```diff
@@ -1,32 +1,44 @@
+import { useMemo } from "react";
+import ScrollableAnchor, { configureAnchors } from "react-scrollable-anchor";
 import s from "./About.module.scss";
+import useWindowSize from "../../utils/customHooks/findWindowSize";
+import setHeightOfHeader from "../../utils/findHeightOfHeader";
 
 export default function About() {
+  const size = useWindowSize();
+
+  const heightOfHeader = useMemo(() => setHeightOfHeader(size.width), [size]);
```
Comment (IgVol) [inline components/About/About.js:10]: Why we find height of header using javascript and not using css?

File: utils/findHeightOfHeader.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+function setHeightOfHeader(windowSize) {
+  let height = -64;
+  if (windowSize >= 1024) {
```
Comment (IgVol) [inline utils/findHeightOfHeader.js:3]: is there any specific reasoning behind selecting this values for windowSize?

**Review by IgVol — CHANGES_REQUESTED** [review]

File: components/About/About.js [hunk 3]
```diff
@@ -1,32 +1,44 @@
+import { useMemo } from "react";
+import ScrollableAnchor, { configureAnchors } from "react-scrollable-anchor";
```
Comment (WebTarantul) [inline components/About/About.js:2]: image

Comment (WebTarantul) [inline components/About/About.js:2]: Navigation does't work.
You can fix it or you can use combination css property  scroll-behavior: smooth; and js functions: scroll or scrollIntoView INSTEAD react scrollable anchor library

**Review by WebTarantul — CHANGES_REQUESTED** [review]

File: components/About/About.module.scss [hunk 4]
```diff
@@ -1,5 +1,9 @@
 @import "../../styles/shared.scss";
 
+/* #navigateToAboutUs {
```
Comment (IgVol) [inline components/About/About.module.scss:3]: delete commented code

**Review by IgVol — CHANGES_REQUESTED** [review]
Looks ok to me, keep codebase clean though

File: components/Header/Header.js [hunk 5]
```diff
@@ -40,28 +42,36 @@ export default function Header() {
...
+                    active: router.pathname === "/map",
+                  })}
+                >
+                  <Link href="/map">
                     <a className={styles.navLink}>карта</a>
                   </Link>
                 </li>
                 <li className={styles.navItem}>
-                  <Link href="">
+                  <Link href="/#navigateToAboutUs">
```
Comment (WebTarantul) [inline components/Header/Header.js:64]: As a developer you want prevent default behavior and use your own function. At the moment default behavior conflicts with your useEffect in a some components.

**Review by WebTarantul — CHANGES_REQUESTED** [review]
To implement correct behavior you need write some function to handle scrolling according params in the history. Then use it at click on the link in the header.
If it's possible to use position sticky for header instead fixed, you can use element.scrollIntoView function and additioanally you don't need use setHeigthOfHeader funciton and it's greate.
 If it is't possible(use position sticky) then you can use scrollTo function, but with correct options.

File: components/Header/Header.js [hunk 6]
```diff
@@ -1,24 +1,88 @@
-import { useState } from "react";
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```
Comment (WebTarantul) [inline components/Header/Header.js:1]: Don't use opportunity like this. Please remove this disable comment and fix any issues according to that rule.

File: components/About/About.module.scss (hunk 4 above)
Comment (WebTarantul) [inline components/About/About.module.scss:3]: Please, delete commented code

**Review by WebTarantul — CHANGES_REQUESTED** [review]

**Review by WebTarantul — APPROVED** [review]

**Review by Dimirolz — APPROVED** [review]

**Pull request merged**